loculus_values/values.yaml

```diff
@@ -25,6 +25,11 @@ secrets:
     clusterWide: "true"
     encryptedData:
       .dockerconfigjson: AgBWKhV1q88KGSfzJq6Dq7qopEjILAWmKY4Wmc3+ryvtvfVkMkx/Tvbyhj8fQ5nerkPfJE+TQVqeBMn1Vln64ZYcw44Nv4yrGLZbpBVRusp6siKvNDPBG+PRnrYYxhuWFovSyCDfjZqIGk8P2URDfrxTTlKzsEOoqEHxQoTwV08ZUlBy+Aqpm9ljjBPLga9gwYkWdjb5vG2DTPOYJ46fsLaFH4wMWdWCY1hgrBXzMV1T12RFH0SXG5+qCnexiyhx98tVt+3bt3tdJYi6AJRC0eQ/OQM0n5OV0HHch+10id5Jy8DMNm25OECwFEp3YO9Er30iLekP7ARvQTAlYwT6OxLjG1zMPtnHGwJB8NiFOsQqZFjupe4pCygdQJU4/wlEYyl3dQvxygwqMDIMx5X3e0iOleJ1JN8VHXzBbWKiy1igujA3Mob4iRC9Stz0jjKOvUSsUbpP54dsU/zyGEKLKUeqWlMuTFL/lmKy/IKZwBnd+98rrZuL6cF0mx6d7TcuVhtvNAVgTp8a8ac7vhJvGF6IZUbLS5D24agZU+RuhOeKPFTvhNgxT8WNCMu4f8gkJPbY6y8qkHcG1oLJQncCSITnXl5tBIIAGijdOf83bX4KEi9uQ2B3l8SjbyAsbZUTZrqfc/7OVJlm2rr0qO2exoIPM6Lq1/06OOtNe483e9CUoBgQTRwKN6807R6uscnrHqy+KdRlhevJCPtxcGiDc7U1MnwuNXnuEnGMbp9KbZDVuPDM9GRXw6SgO3t48im17npls1Lj6CNys26paqM4qQQ/pwX14LMoNVigFlhkSaK+M400AQaYlpV3D5LRVzD2FJzB9AO9JsPEayifUgjNl8gezoEg867c/Cpx4Xf18jzahTHzMGbs44POATM=
+  basicauth:
+    type: sealedsecret
+    clusterWide: true
+    encryptedData:
+      users: AgBfZyTygLwse/CBQx+qy2tY+pfcFulezoyqFOofNu/ZkKfIokObxiUYPVKBHwn2QavQgU7TK//NNLylADNUVwi5rWbRP0akOJYLb48h8GaRF3dyRp6t4OhB3N0obuaKmVyYW0C31AzabdLfB7//E4Xzg/TWbaz+p7vgYD9SPNb4/Qk1svDoLNKOi10Ghdi8zR8YB5Sje5p+ah0to21K3YfHdg68y13AirIwcQqo1PmpO9jcbwSy9y00A71vffnmYMXIqBLppriPnFs/4p/OtBKz73eRE7GcRkNt1lfxmiv5uOB24tBDHbTENpzGeDqdWLCPu7aa5Vwt6JQFz4pXtWVVPpsxRYtRGx96uadoLG3Za5fN66TADUEatxvNDbiWv+421qwdgogFYvEa7LWISkwKI2Hm6/wpyOJysaLKzPl8159l/tavXQyphMamoaxiqDNPyDYJ/cbMV4TUBMd80zfQMep2PPcnT9oxJcMFKeFJSEkEgJ2LCE58Wrp/N01FPvmp5rQDrTCAvqJE4xL3TepXoGEBzgYN2coF5ChL4r1wdCd/3l6PdV6mLHanu26xJ7XEF8/fKOX1pCtNcIBQGvr2pX3NTii54Sp0b6YvDpgCdi2S0Q1mjqwG4ZZNkvrj1rH7KigxPtl+1k/4Pz8a2X/JMq5rubXCa+QzsAxZZtlK2hRa1CrA0/f3nZdX7jHIpSS7H5H48QF7CpWghq/SwE9jgk74z/xwweSrpp33CPTTFKmgE5PGc4iD
```

Comment:
Can you explain what is inside the encrypted stuff (except for the literal secret of course) - this looks a bit cryptic to me!

Reply:
Made by https://httpd.apache.org/docs/2.4/programs/htpasswd.html - not 100% sure if it will work

Reply:
Ok I get it now. Basic auth middleware expects a spec like this:

```
apiVersion: traefik.containo.us/v1alpha1
kind: Middleware
metadata:
  name: basic-auth
spec:
  basicAuth:
    users: admin:$$apr1$$xxxxxxxx$$xx.xxxxxx/xxxxx/xxx
```
where `admin...` is an apache htpasswd string that encode username and password.

Now rather than literally encoding it like this, we use a secret to sub in this keyval pair `users: admin:$$apr1$$xxxxxxxx$$xx.xxxxxx/xxxxx/xxx`.

We could have done this using a raw secret:

```
  basicauth:
    type: raw
    data:
      users: admin:$$apr1$$xxxxxxxx$$xx.xxxxxx/xxxxx/xxx
```

but of course then it's not secret.

What we did for testing was more convenient but essentially the same:

```
  basicauth:
    type: rawhtpasswd
    data:
      username: loculus
      password: widetailpotato
```

which gets transformed into exactly the same string as the equivalent raw one, just more convenient as one doesn't need to run through `htpasswd`.

However we can't do this with a sealed secret of course.

So now we need to seal the original htpasswd encoded string. Which is what you're doing here.